### PR TITLE
NMA-681: DashPay:  Send to contact (flow)

### PR DESCRIPTION
**Pull request by HashEngineering**
## Issue being fixed or feature implemented

NMA-681

## Related PR's and Dependencies

N/A
## Screenshots / Videos

None, yet.
## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

File: wallet/src/de/schildbach/wallet/ui/dashpay/BottomNavFragment.kt [hunk 1]
```diff
@@ -33,9 +37,28 @@ abstract class BottomNavFragment(@LayoutRes contentLayoutId: Int) : Fragment(con
...
 
+    /*fun showNavigation(show: Boolean) {
+        val navParentView = mainActivity.bottom_navigation.parent.parent
+        if (navParentView is KeyboardResponsiveCoordinatorLayout) {
+            navParentView.forceHideViewToHide = !show
+        } else {
+            mainActivity.bottom_navigation.visibility = if (show) View.GONE else View.VISIBLE
+        }
+    }*/
+
```
Comment (tomasz-ludek) [inline wallet/src/de/schildbach/wallet/ui/dashpay/BottomNavFragment.kt:48]: This can be removed

**Review by tomasz-ludek — CHANGES_REQUESTED** [review]
One minor thing to improve.

File: wallet/src/de/schildbach/wallet/ui/dashpay/BottomNavFragment.kt (hunk 1 above)
Comment (HashEngineering) [reply wallet/src/de/schildbach/wallet/ui/dashpay/BottomNavFragment.kt]: Removed https://github.com/dashevo/dash-wallet/pull/598/commits/95e566386798a689329efecdb02b68b91d71480f

**Review by tomasz-ludek — APPROVED** [review]
Looks good!

**Pull request merged**